src/test/datascience/interactiveWindow.vscode.common.test.ts

```diff
@@ -371,11 +372,27 @@ ${actualCode}
         assert.equal(notebookDocument?.cellCount, 3, `Running a whole file did not split cells`);
 
         // Make sure it output something
-        notebookDocument?.getCells().forEach((c, i) => {
-            if (c.document.uri.scheme === 'vscode-notebook-cell' && c.kind == vscode.NotebookCellKind.Code) {
-                assertHasTextOutputInVSCode(c, `${i}`);
-            }
-        });
+        let lastErrorMessage = 'No output found';
+        await waitForCondition(
```

Comment:
Found a flaky test, we weren't waiting for the output.
we need to wait for the output.

Reply:
huh, I imagine this would cause other tests to be flaky as well

Reply:
I think most of the other tests do have this, i know for sure that all the nb tests have this as i wrote/updated most of them.
IW is somethign thats been around for a while and I haven't really touched that area much...